src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingService.kt

```diff
@@ -166,6 +170,102 @@ class VurderingService(
         tilbakestillEndretTidForKopierteVurderinger(kopiAvVurderinger, tidligereVurderinger)
     }
 
+    @Transactional
+    fun gjenbrukInngangsvilkårVurderinger(
+        nåværendeBehandlingId: UUID,
+        tidligereBehandlingId: UUID,
+    ) {
+        validerBehandlingForGjenbruk(
+            nåværendeBehandlingId,
+            tidligereBehandlingId
+        )
+        val forrigeVurderingIdTilNåværendeBarnMap =
+            finnBarnPåBeggeBehandlinger(tidligereBehandlingId, nåværendeBehandlingId)
+        val sivilstandErLik =
+            hentGrunnlagsdataOgValiderSivilstand(tidligereBehandlingId, nåværendeBehandlingId)
+        val (tidligereVurderinger, nåværendeVurderinger) = finnVurderingerSomSkalGjenbrukes(
+            tidligereBehandlingId,
+            sivilstandErLik,
+            forrigeVurderingIdTilNåværendeBarnMap,
+            nåværendeBehandlingId
+        )
+
+        val vurderingerSomSkalLagres = tidligereVurderinger.mapNotNull { tidligereVurdering ->
+            if (tidligereVurdering.type == VilkårType.ALENEOMSORG) {
```

Comment:
Denne gjelder kanskje `ALDER_PÅ_BARN` også?
```suggestion
            if (tidligereVurdering.type.gjelderFlereBarn()) {
```

Reply:
Alder på barn er et aktivitetsvilkår, så den skal ikke kunne gjenbrukes. Men det blir jo ikke feil heller, siden aktivitetsvilkår allerede er luket ut